src/App.css

```diff
@@ -36,3 +36,131 @@
     transform: rotate(360deg);
   }
 }
+
+.wrapper {
+  max-width: 1200px;
+  width: 90%;
+  margin: 0 auto;
+}
+
+.progressBar {
+  height: 5px;
+  background-color: #8AAAE5;
+  margin: 20px 0;
+  border-radius: 5%;
+  box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+}
+
+h1 {
+  color: #8AAAE5;
+}
+html {
+  background-color: #424651;
+  color: whitesmoke;
+}
+
+.imgContainer {
```

Comment:
Add margin, set width and height: 

Suggestion 

.imgContainer {
  width: 400px;
  height: 300px;
  margin: 25px;
}